<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPasswordWidget</class>
 <widget class="QWidget" name="ForgotPasswordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1271</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Reset Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Input your Email :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="leEmail">
      <property name="placeholderText">
       <string>Input your Email</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Input your new password :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="leNewPassword">
      <property name="echoMode">
       <enum>QLineEdit::NoEcho</enum>
      </property>
      <property name="placeholderText">
       <string>Input new password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="leConfirmPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password Confirmation</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btnResetPassword">
      <property name="text">
       <string>Reset Password</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
